### PR TITLE
Command system

### PR DESCRIPTION
**Pull request by Agtheo09**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

**Comment by Agtheo09** [comment]
Merging working command_base branch to master

**Comment by Agtheo09** [comment]
@GrBrogremmer09 @alex-bene

**Review by geoRafto — APPROVED** [review]

File: TeamCode/FreightFrenzyAutonomousPlanA_NoOtherSide.java
Comment (alex-bene) [inline TeamCode/FreightFrenzyAutonomousPlanA_NoOtherSide.java:1]: this file should be deleted

File: TeamCode/FreightFrenzyAutonomousPlanA_NoOtherSide_Inverted.java
Comment (alex-bene) [inline TeamCode/FreightFrenzyAutonomousPlanA_NoOtherSide_Inverted.java:1]: this file should be deleted

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PowerPlayTeleOp.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        Slidy = new SlidyRobot(hardwareMap, RobotConstants, telemetry, driverOp, toolOp);
+    }
+
+//    @Override
+//    public void run() {
+//        super.run();
+//        // TODO: Make telemetry subsystem/command and remove this function
+//        Slidy.telemetryUpdate();
+//        Slidy.dashboardTelemetryUpdate();
+//    }
```
Comment (alex-bene) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PowerPlayTeleOp.java]: why is this commented out? Do we have the telemetry subsystem?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/AprilTagDetectionSubsystemInverted.java
Comment (alex-bene) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/AprilTagDetectionSubsystemInverted.java:1]: why do we even need this?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/SlidyRobot.java
Comment (alex-bene) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/SlidyRobot.java:1]: whatever code we do not use, please delete it (don't just comment it out).
The only exception would be the front-slider related code which you can just add a note (this is implemented but we ended up not using it for the competitions due to mechanical bugs)

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/AutonomousCommand.java
Comment (alex-bene) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/AutonomousCommand.java:1]: delete?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/ElevatorManualCommand.java
Comment (alex-bene) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/ElevatorManualCommand.java:1]: if we don't have manual controls, we do not need manual command

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/SimpleAuto.java
Comment (alex-bene) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/SimpleAuto.java:1]: the simple auto name here might not be very good

**Review by alex-bene — CHANGES_REQUESTED** [review]
In total, every file we ended up NOT using should be deleted except the front-slider-related ones we should keep. But, even then, there might be some at places more than on files about the same thing for the front slider (so some should go). The same thing goes for code comments (not explanations -- just code) that we do not use (again with the exception of the front slider).

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PowerPlayTeleOp.java (hunk 1 above)
Comment (Agtheo09) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PowerPlayTeleOp.java]: I created a telemetry subsystem that runs telemetr.update() in the periodic()!

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/AprilTagDetectionSubsystemInverted.java
Comment (Agtheo09) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/AprilTagDetectionSubsystemInverted.java:1]: I believe that the IDS are different to make different parking positions but @GrBrogremmer09 was in charge of writing this file

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/ElevatorManualCommand.java
Comment (Agtheo09) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/ElevatorManualCommand.java:1]: We do have manual controls

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/AutonomousCommand.java
Comment (Agtheo09) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/AutonomousCommand.java:1]: Yes

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/SimpleAuto.java
Comment (Agtheo09) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Slidy_PPV2/commands/SimpleAuto.java:1]: Ok New Name: AutoOneConeCommand

**Comment by Agtheo09** [comment]
@alex-bene I resolved every requested changes from you in the next 2 commits after the pull request!

**Comment by alex-bene** [comment]
@Agtheo09  there are still remaining code-comments that we do not need. Also, let's rename Slidy_PPV2 to just powerplayrobot (or something similar) and slidyrobot to PowerPlayRobot (let's keep the slidy names etc for the notebook and internal use and not the code). Similarly, PPparking.java --> PowerPlayAutonomousParking.java

**Comment by Agtheo09** [comment]
@alex-bene I removed(hopefully) all the unnecessary comments and renamed the robot!

**Review by alex-bene — APPROVED** [review]

**Pull request merged**